Extensions/xkit_preferences.js

```diff
@@ -74,12 +76,40 @@ XKit.extensions.xkit_preferences = new Object({
 			XKit.tools.add_css(mobile_control_panel, 'mobile_xkit_menu');
 		}
 
+		const react_add_button = async (button) => {
+			const account_label = await XKit.interface.translate("Account");
+			const menu_label = await XKit.interface.translate("Menu");
+
+			const insert = () => {
+				if (button.isConnected) return;
+				const header = document.querySelector('#base-container > div > div > header');
+				if (header === null) return;
+
+				// desktop
+				const accountButton = header.querySelector(`[aria-label="${account_label}"]`);
```

Comment:
```suggestion
				const desktopAccountButton = header.querySelector(`[aria-label="${account_label}"]`);
```

The Primary Function Of Code Is To Be Read By Humans